src/main/java/io/jenkins/plugins/monitoring/MonitorPortlet.java

```diff
@@ -56,5 +66,8 @@
      *          (e.g. http://localhost:8080/jenkins/job/job-name/view/change-requests/job/PR-1/1/),
      *          or {@code Optional.empty()}, if no link should be added to portlet.
      */
-    Optional<String> getDetailViewUrl();
+    public Optional<String> getDetailViewUrl() {
```

Comment:
The JavaDoc should use a relative URL as an example. You can additionally show the absolute URL.